lib/fauna/query.rb

```diff
@@ -91,7 +105,8 @@ def var(name)
     #
     # Reference: {FaunaDB Basic Forms}[https://faunadb.com/documentation/queries#basic_forms]
     def if_(condition, then_, else_)
-      { if: condition, then: then_, else: else_ }
+      # rubocop:disable Style/HashSyntax
```

Comment:
I'm not actually sure how rubocop does scoping. This comment disables the warning raised both here and in `do`. Adding it in both places doesn't work.